api/health/health.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+
+package health;
+
+option go_package = "github.com/talos-systems/talos/api/health";
+
+import "google/protobuf/empty.proto";
+
+service Health {
+  rpc Check(google.protobuf.Empty) returns (HealthCheckResponse);
+  rpc Watch(HealthWatchRequest)  returns (stream HealthCheckResponse);
+  rpc Ready(google.protobuf.Empty) returns (ReadyCheckResponse);
+}
+
+message HealthWatchRequest {
+	int64 count = 1;
+  int64 interval_seconds = 2;
```

Comment:
nit: these are misaligned

Reply:
I don't think we should have a count, but rather the client should decide when its done and should watch forever.

Reply:
by default it works like that. a 0 value indicates forever, >0 indicates a limit to the retries. 

ping vs ping -c4 for example.

Reply:
ahh cool.